lib/v2/screens/transfer/send_search_user/send_search_user_screen.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/v2/components/search_user/search_user_widget.dart';
+import 'package:seeds/v2/datasource/remote/model/member_model.dart';
+import 'package:seeds/v2/screens/transfer/send_search_user/interactor/send_search_user_bloc.dart';
+import 'package:seeds/v2/screens/transfer/send_search_user/interactor/viewmodels/send_search_user_state.dart';
+
+/// SendScannerScreen SCREEN
+class SendSearchUserScreen extends StatefulWidget {
+  @override
+  _SendSearchUserScreenState createState() => _SendSearchUserScreenState();
+}
+
+class _SendSearchUserScreenState extends State<SendSearchUserScreen> {
+  late SearchUserWidget _searchUserWidget;
+  final _sendSearchUserPageBloc = SendSearchUserPageBloc();
+
+  @override
+  void initState() {
+    super.initState();
+    _searchUserWidget = SearchUserWidget(resultCallBack: onResult);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider<SendSearchUserPageBloc>(
+        create: (_) => _sendSearchUserPageBloc,
+        child: BlocBuilder<SendSearchUserPageBloc, SendSearchUserPageState>(
```

Comment:
BlocBuilder is not needed here, there is nothing to update at UI level.

Reply:
Not on this PR, but next pr users it. Ill remove until next pr tho